tools/pflua-compile

```diff
@@ -40,12 +40,18 @@ if (not(flags["--bpf-asm"] or flags["--bpf-lua"] or flags["--lua"])) then
    flags["--lua"] = true
 end
 
+
+local filter disable_optimization = false
+if flags["-O"] then disable_optimization = true end
```

Comment:
Since you ask :) I'd prefer -O0 to indicate no optimization, as in GCC, and a bare -O to be a synonym of -O1.